datashader/reductions.py

```diff
@@ -193,7 +199,10 @@ def apply(self, df):
             else:
                 nullval = 0
             a = cupy.asarray(a)
-            b = cupy.asarray(df[self.column].to_gpu_array(fillna=nullval))
+            if cudf.__version__ >= "22.02":
```

Comment:
I think this needs to be something like `packaging.version.Version("22.02")`.

Reply:
fixed it, thanks :)